src/editors/containers/VideoEditor/components/VideoSettingsModal/components/VideoSourceWidget/index.jsx

```diff
@@ -144,12 +143,6 @@ export const VideoSourceWidget = ({
 VideoSourceWidget.propTypes = {
   // injected
   intl: intlShape.isRequired,
-  // redux
-  updateField: PropTypes.func.isRequired,
 };
-export const mapStateToProps = () => ({});
 
-export const mapDispatchToProps = (dispatch) => ({
```

Comment:
Do you mean making this component no longer "connected?"

Reply:
Yes, from what I understand, you shouldn't need to connect via redux-thunk when you use the useSelector hook.